app/javascript/pages/form/components/AlternateContactType.scss

```diff
@@ -0,0 +1,9 @@
+.radio-field-group {
+  margin-top: var(--seeds-spacer-label);
+  display: flex;
+  flex-direction: column;
+}
+
+.field {
+  margin-bottom: var(--seeds-spacer-label);
+}
```

Comment:
Seems like these should be in a global style sheet, since they'll apply to all fields, right?  They're not specific to this component.  Also seems like we'd want a RadioGroup component at some point to make this reusable.